### ISSUE TITLE
Check if modal has been closed once or destroyed

### ISSUE THREAD

**Issue by cvanderlinden** [VodkaBears/Remodal#284]
I am looking for a way to see if the modal has already been closed once or .destroy() has been called.

I got around it by adding a classed on modal close, and the checking if the modal exists with that class, it's a little hacky.  Wondering if there is a better option here.

**Comment by yunusga** [comment]
Check it in [Remodal#methods](https://github.com/vodkabears/Remodal#methods) and [Remodal#events](https://github.com/vodkabears/Remodal#events)

**Comment by cvanderlinden** [comment]
Right, I did.  getState() can tell me when it's closed, but it will also say closed before it's been activated once.  Events is what I use to add the CSS class I mentioned in the first comment.  Doesn't help.
